gallery/src/styles/NxList/NxListBulletedExample.tsx

```diff
@@ -11,15 +11,40 @@ const NxListWithBullets = () =>
     <h4 className="nx-list__title">
       List title
     </h4>
-    <ul>
+    <ul className="nx-list nx-list--bulleted">
       <li className="nx-list__item">
         item 1
       </li>
       <li className="nx-list__item">
         Item 2
```

Comment:
Mixing text nodes and elements as siblings

Reply:
> Mixing text nodes and elements as siblings

Bare nodes now wrapped with `<span>`'s: https://github.com/sonatype/sonatype-react-shared-components/pull/18/commits/08dfe7f04fdd3ffabf59b3d45d6adab914c0dc84